lib/resolveurl/plugins/mixdrop.py

```diff
@@ -33,7 +33,7 @@ def get_media_url(self, host, media_id):
         try:
             r = re.search('\s+?(eval\(function\(p,a,c,k,e,d\).+)\s+?', html)
             r = jsunpack.unpack(r.group(1))
-            r = re.search('vsrc="([^"]+)', r.replace('\\', ''))
+            r = re.search('vsrc?="([^"]+)', r.replace('\\', ''))
```

Comment:
can you explain what you are doing here as adding a `?` looks for `vsrc=` or `vsr=`

Reply:
Both.
I found links which require 'vsr=...'.
I don't know if this is a general rule or not, so i put a question mark after 'vsrc' to match both 'vsr' and 'vsrc'.